src/pages/NoGrants.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, Typography } from '@mui/material';
+import FullPageDialog from 'components/fullPage/Dialog';
+import { useClient } from 'hooks/supabase-swr';
+import useBrowserTitle from 'hooks/useBrowserTitle';
+import { FormattedMessage } from 'react-intl';
+
+const NoGrants = () => {
```

Comment:
Full page dialog that let's user know we'll reach out to them soon after reviewing their sign up. Included a log out button so the user does not get stuck on that page.